relationships/candidates/AWSCLOUDFRONT.yml

```diff
@@ -0,0 +1,20 @@
+category: AWSCLOUDFRONT
+lookups:
+  - entityTypes:
+      - domain: INFRA
+        type: AWSCLOUDFRONT
+    tags:
+      matchingMode: ALL
+      predicates:
+        - tagKeys: ["aws.region", "aws.awsRegion"]
+          field: region
+        - tagKeys: ["aws.accountId"]
```

Comment:
When only domainName is used in relationship files, do we need region and accountId to be added here?

Reply:
Can't comment, Will check with entity-platform team on this.

Reply:
We don't have knowledge about AWS services and cannot help you with what fields should be used for your definitions. Entity Platform can only help you with the structure of the definitions.

You need to validate what is required to identify an `AWSCLOUDFRONT` instances uniquely.

Do you expect multiple CloudFront instances having the same `aws.region` and `aws.cloudfront.domainName` but a distinct `aws.accountId`? If not, you can safely remove the `aws.accountId` field as @nr-sparsi advised.

Reply:
Bucket Names are unique across regions and accounts, hence removed account and region.